src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (repairedRobot.hasOwnProperty(value)) {
+      return null;
+    }
+
+    repairedRobot[value] = key;
+  }
+
+  return repairedRobot;
```

Comment:
```suggestion
  const repairedRobot = {};

  for (const key in robot) {
    if (repairedRobot.hasOwnProperty(robot[key])) {
      return null;
    }

    repairedRobot[robot[key]] = key;
  }

  return repairedRobot;
```

you don't have to use this variable, but I'm not sure that it's important